.editorconfig

```diff
@@ -1,3 +1,4 @@
 [{*.kt,*.kts}]
-disabled_rules = no-wildcard-imports
-ij_kotlin_imports_layout = *
\ No newline at end of file
+indent_size = 4
```

Comment:
Recent version of [kotlinter](https://github.com/jeremymailen/kotlinter-gradle) doesn't have indent setting anymore, hence I moved it from `build.gradle.kts` to `.editorconfig`.